app/static/js/events/scheduler.js

```diff
@@ -259,18 +260,31 @@ function addSessionToTimeline(sessionRef, position, shouldBroadcast) {
     updateColor(sessionRefObject.$sessionElement, sessionRefObject.session.track);
 
     var $mobileSessionElement = $(mobileSessionTemplate);    
-    var content = sessionRefObject.session.title + " | ";
-    var speakers = [];
-    _.each(sessionRefObject.session.speakers, function(speaker) {
-        speakers.push(speaker.name);
-    });
-    content += speakers.join(', ');
     $mobileSessionElement.find('.time').text(sessionRefObject.session.start_time.format('hh:mm A'));
-    $mobileSessionElement.find('.event').text(content);
-    $mobileSessionElement.find('.event').attr("data-target", "#session-track-details"+sessionRefObject.session.id);
+    $mobileSessionElement.find('.title').text(sessionRefObject.session.title);
     $mobileSessionElement.find('.session-track-details').attr("id", "session-track-details"+sessionRefObject.session.id);
-    $mobileSessionElement.find('.session-speakers').text("Speakers: " + speakers.join(', '));
+    $mobileSessionElement.find('.event').attr("data-target", "#session-track-details"+sessionRefObject.session.id + ",#session-track-short-details"+sessionRefObject.session.id);
```

Comment:
There are two collapsible elements here. So the two ids refer to them. One of them shows the shorter version of the details of the speakers(it is visible by default). The other one is the longer version of the speaker details which is hidden by default. When we click on the container, the former one becomes hidden and the latter one becomes visible.